packages/manager/src/features/CloudPulse/Alerts/CreateAlert/CreateAlertDefinition.tsx

```diff
@@ -0,0 +1,262 @@
+  //   const notificationTemplateList = newNotifications.map(
+  //     (notification) => notification.id
+  //   );
+  //   setValue('channel_ids', notificationTemplateList);
+  //   setNotifications(newNotifications);
+  //   setOpenAddNotification(false);
+  // };
+
+  const onSubmit = handleSubmit(async (values) => {
+    try {
+
+      await createAlert(values);
+      enqueueSnackbar('Alert successfully created', {
+        variant: 'success',
+      });
+      alertCreateExit();
+    } catch (errors) {
+      for (const error of errors) {
+        if (error.field) {
+          setError(error.field, { message: error.reason });
+        } else {
+          setError('root', { message: error.reason });
+        }
+      }
+    }
+  });
+
+  const { newPathname, overrides } = generateCrumbOverrides();
+
+  const engineOptionValue = watch('service_type');
+  return (
+    <Paper>
+      <Breadcrumb
+        crumbOverrides={overrides}
+        pathname={newPathname}
+      ></Breadcrumb>
+      <FormProvider {...formMethods}>
+        <form onSubmit={onSubmit}>
+          <Typography variant="h2">1. General Information</Typography>
+          <Controller
+            render={({ field, fieldState }) => (
+              <TextField
+                data-testid="alert-name"
+                errorText={fieldState.error?.message}
+                label="Name"
+                name={'label'}
+                onBlur={field.onBlur}
+                onChange={(e) => field.onChange(e.target.value)}
+                value={field.value ?? ''}
+              />
+            )}
+            control={control}
+            name="label"
+          />
+          <Controller
+            render={({ field, fieldState }) => (
+              <TextField
+                errorText={fieldState.error?.message}
+                label="Description"
+                name={'description'}
+                onBlur={field.onBlur}
+                onChange={(e) => field.onChange(e.target.value)}
+                optional
+                value={field.value ?? ''}
+              />
+            )}
+            control={control}
+            name="description"
+          />
+          <CloudPulseServiceSelect name="service_type" />
+          {engineOptionValue === 'dbaas' && (
+            <EngineOption
+              engineOptions={engineOptions ?? []}
+              isError={!!engineOptionError}
+              isLoading={engineOptionLoading}
+              name={'engineOption'}
+            />
+          )}
+          <CloudPulseRegionSelect name="region" />
+          <CloudPulseMultiResourceSelect
+            engine={engineOptionValue}
+            name="resource_ids"
+            region={watch('region')}
+            serviceType={watch('service_type')}
+          />
+          <CloudPulseAlertSeveritySelect name="severity" />
+          <MetricCriteriaField
+            getMaxInterval={(interval: number) =>
+              setMaxScrapeInterval(interval)
```

Comment:
here the getMaxInterval prop name seems confusing because it sets the maxScrapeInterval.

Reply:
changing it to setMaxInterval